README.md

```diff
@@ -1,10 +1,10 @@
-# virt-ui
+# forklift-ui
 
 Migration Toolkit for Virtualization UI
 
-[![Build Status](https://travis-ci.com/konveyor/virt-ui.svg?branch=master)](https://travis-ci.com/konveyor/virt-ui)
+[![Build Status](https://travis-ci.com/konveyor/forklift-ui.svg?branch=master)](https://travis-ci.com/konveyor/forklift-ui)
```

Comment:
Can you actually just remove this Travis badge? It won't work anymore now anyway, we can figure out how to put a GH Actions one in later